src/calendars/ireland.ts

```diff
@@ -38,7 +38,7 @@ const dates = async (config: Config): Promise<Array<RomcalDateItemInput>> => {
       date: dayjs.utc(`${year}-1-30`),
     },
     {
-      key: 'saintBridgetOfSwedenReligious',
+      key: 'saintBrigidVirgin',
```

Comment:
Why are you reducing its name, like a few other. She is well know as Bridget of Sweden, so it's still useful to keep this information in his key.

Reply:
[St Bridget of Sweden](https://en.wikipedia.org/wiki/Bridget_of_Sweden) and [St Brigid of Kildare](https://en.wikipedia.org/wiki/Brigid_of_Kildare) are two different saints.

Reply:
Understood, this isn't the same person.